netsim/roles/bridge.py

```diff
@@ -0,0 +1,110 @@
+'''
+Bridge-specific data transformation:
+
+* Define 'br_default' VLAN 1 if it's not defined
+* Set vlan.mode to 'bridge' on bridge devices
+* Set vlan.access on bridge interfaces without the 'vlan' parameter
+'''
+import typing
+
+from box import Box, BoxList
+
+from ..utils import log
+from ..data import global_vars,append_to_list,get_box
+from ..augment import links
+from . import select_nodes_by_role
+
+"""
+Set the default VLAN mode on bridges to 'bridge' in the pre-transform hook
+to prevent VLAN module from setting it to some other value.
+
+Also, define 'br_default' node VLAN if it's not defined
+
+Return 'True' if we found at least one bridge (so we need further processing)
+"""
+def create_default_VLAN(topology: Box) -> bool:
+  BR_DEFAULT = global_vars.get_const('bridge.default_vlan.name','br_default')
```

Comment:
I would expect the default VLAN name and ID to be part of the VLAN module, rather than this 'bridge' role

Reply:
This is not a VLAN default. This is the default used for the devices with **bridge** role.